src/Access/IAccessStorage.cpp

```diff
@@ -418,9 +420,20 @@ UUID IAccessStorage::login(
     const String & user_name,
     const String & password,
     const Poco::Net::IPAddress & address,
-    const ExternalAuthenticators & external_authenticators) const
+    const ExternalAuthenticators & external_authenticators,
+    bool log_and_mask_exceptions) const
```

Comment:
The name `log_and_mask_exceptions` doesn't seem to be clear. May be we should rename it to something like `replace_exception_with_cannot_authenticate`.

Reply:
Changed.